src/components/PlatformNavigator/__tests__/PlatformNavigator.test.tsx

```diff
@@ -14,9 +14,13 @@ const routerMock = {
 
 jest.mock('next/router', () => routerMock);
 
+const flatDirectoryMock = {};
+
+jest.mock('@/directory/flatDirectory.json', () => flatDirectoryMock);
+
 describe('PlatformNavigator', () => {
   const component = (
-    <PlatformNavigator currentPlatform={'react'} isPrev={true} />
```

Comment:
This is just fixing some type errors (`isPrev` prop no longer exists on PlatformNavigator, but `isGen1` is requried)